cmake/modules/AddSwift.cmake

```diff
@@ -1774,6 +1787,18 @@ function(_add_swift_executable_single name)
     list(APPEND link_flags "-fuse-ld=gold")
   endif()
 
+  INCLUDE(CheckCXXCompilerFlag)
```

Comment:
I don't think this block is necessary.  This function is used to configure compiler executables, which don't have any Swift code in them.